hw/ip/hmac/dv/hmac_sim_cfg.hjson

```diff
@@ -0,0 +1,93 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  // Name of the sim cfg - typically same as the IP under test.
+  name: hmac
+
+  // Top level dut name (sv module).
+  dut: hmac
+
+  // Top level testbench name (sv module).
+  tb: tb
+
+  // Fusesoc core file used for building the file list.
+  fusesoc_core: lowrisc:dv:hmac_sim:0.1
+
+  // Testplan hjson file.
+  testplan: "{proj_root}/hw/ip/hmac/data/hmac_testplan.hjson"
+
+  // RAL spec - used to generate the RAL model.
+  ral_spec: "{proj_root}/hw/ip/hmac/data/hmac.hjson"
+
+  // Import additional common sim cfg files.
+  import_cfgs: [// Project wide common sim cfg file
+                "{proj_root}/hw/dv/data/common_sim_cfg.hjson",
+                // Common CIP test lists
+                "{proj_root}/hw/dv/data/tests/csr_tests.hjson",
+                "{proj_root}/hw/dv/data/tests/intr_test.hjson",
+                "{proj_root}/hw/dv/data/tests/tl_access_tests.hjson",
+                "{proj_root}/hw/dv/data/tests/stress_tests.hjson"]
```

Comment:
this is awesome. So i guess if we have a module that for some reason can't run a specific version (maybe it's a standalone module that doesn't have a tl interface for some reason), we could just choose exactly which to import?

Reply:
Yes. These are common tests that are grouped together - they are closely tied to our CIP DV framework. The notion of 'importing' sets of tests makes it easy and quick to build the hjson based test infrastructure for an IP. Depending on features an IP has, it can pick and choose to enable these common tests. There is a 'mem_tests.hjson' that UART doesnt import, but SPI and other IPs that contain memories will.